drivers/fuel_gauge/max17048/max17048.c

```diff
@@ -39,17 +39,14 @@ struct max17048_data {
 	bool charging;
 };
 
-/* Read buffer */
-unsigned char max17048_buffer[2];
-
 /**
  * I2C communication
  * The way we read a value is first writing the address we want to read and then
  * wait for 2 bytes containing the data.
  */
 int max17048_read_register(const struct device *dev, uint8_t registerId, uint16_t *response)
 {
-
+	unsigned char max17048_buffer[2];
```

Comment:
Bah, I should've caught this earlier sorry. Can we make this a `uint8_t` array? That seems to be the style I've seen in Zephyr.

Reply:
It's effectively the same, so I don't think there's any reason to retest.